typescript/phoenix-sdk/src/client.ts

```diff
@@ -804,6 +793,457 @@ export class Client {
+    args: PlaceMultiplePostOnlyOrdersInstructionArgs,
+    marketAddress: string,
+    trader?: PublicKey
+  ): TransactionInstruction {
+    if (!trader) {
+      trader = this.trader.pubkey;
+    }
+    const market = this.markets.get(marketAddress);
+    if (!market) throw new Error("Market not found: " + marketAddress);
+    return market.createPlaceMultiplePostOnlyOrdersInstruction(args, trader);
+  }
+
+  /**
+   * Creates a _PlaceMultiplePostOnlyOrdersWithFreeFunds_ instruction.
+   *
+   * @param args to provide as instruction data to the program
+   * @param marketAddress Market address string
+   * @param trader Trader public key
+   *
+   * @category Instructions
+   */
+  public createPlaceMultiplePostOnlyOrdersInstructionWithFreeFunds(
+    args: PlaceMultiplePostOnlyOrdersInstructionArgs,
+    marketAddress: string,
+    trader?: PublicKey
+  ): TransactionInstruction {
+    if (!trader) {
+      trader = this.trader.pubkey;
+    }
+    const market = this.markets.get(marketAddress);
+    if (!market) throw new Error("Market not found: " + marketAddress);
+    return market.createPlaceMultiplePostOnlyOrdersInstructionWithFreeFunds(
+      args,
+      trader
+    );
+  }
+
+  /**
+   * Creates a _ReduceOrder_ instruction.
+   *
+   * @param args to provide as instruction data to the program
+   * @param marketAddress Market address string
+   * @param trader Trader public key
+   *
+   * @category Instructions
+   */
+  public createReduceOrderInstruction(
+    args: ReduceOrderInstructionArgs,
+    marketAddress: string,
+    trader?: PublicKey
+  ) {
+    if (!trader) {
+      trader = this.trader.pubkey;
+    }
+    const market = this.markets.get(marketAddress);
+    if (!market) throw new Error("Market not found: " + marketAddress);
+    return market.createReduceOrderInstruction(args, trader);
+  }
+
+  /**
+   * Creates a _ReduceOrderWithFreeFunds_ instruction.
+   *
+   * @param args to provide as instruction data to the program
+   * @param marketAddress Market address string
+   * @param trader Trader public key
+   *
+   * @category Instructions
+   */
+  public createReduceOrderWithFreeFundsInstruction(
+    args: ReduceOrderWithFreeFundsInstructionArgs,
+    marketAddress: string,
+    trader?: PublicKey
+  ) {
+    if (!trader) {
+      trader = this.trader.pubkey;
+    }
+    const market = this.markets.get(marketAddress);
+    if (!market) throw new Error("Market not found: " + marketAddress);
+    return market.createReduceOrderWithFreeFundsInstruction(args, trader);
+  }
+
+  /**
+   * Creates a _RequestSeat_ instruction.
+   *
+   * @param marketAddress Market address string
+   * @param payer Payer public key
+   * @param trader Trader public key
+   *
+   * @category Instructions
+   */
+  public createRequestSeatInstruction(
+    marketAddress: string,
+    payer?: PublicKey,
+    trader?: PublicKey
+  ) {
+    if (!trader) {
+      trader = this.trader.pubkey;
```

Comment:
not `trader = payer`?

Reply:
![image](https://user-images.githubusercontent.com/61092285/235468335-5690f3f6-141f-4603-b710-144839ed9931.png)
changed to this